<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="Collect" name="tab_collect">
       <attribute name="title">
        <string>Collect</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_align">
       <attribute name="title">
        <string>Align</string>
       </attribute>
      </widget>
      <widget class="Colocalize" name="tab_coloc">
       <attribute name="title">
        <string>Colocalize</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1313</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import microscopy data</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save session</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear session</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Collect</class>
   <extends>QWidget</extends>
   <header>impose.gui.collect</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Colocalize</class>
   <extends>QWidget</extends>
   <header>impose.gui.colocalize</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
